dependencies.gradle

```diff
@@ -9,13 +9,13 @@ ext.versions = [
 
 def gradle = "7.0.4"
 // Ref: https://kotlinlang.org/releases.html
-def kotlin = "1.5.31"
-def kotlinCoroutines = "1.5.2"
+def kotlin = "1.6.0"
```

Comment:
hopefully the update to 1.6.10 will be much simpler

Reply:
Yes, I have seen that 1.6.10 is available, but I prefer dependabot to do his work :)